frappe/public/js/frappe/form/sidebar/form_sidebar.js

```diff
@@ -177,10 +177,10 @@ frappe.ui.form.Sidebar = Class.extend({
 			parent: this.sidebar.find(".form-shared")
 		});
 	},
-	make_viewers: function() {
-		this.frm.viewers = new frappe.ui.form.Viewers({
+	make_sidebar_users: function() {
```

Comment:
`make_viewers` seemed fine, any particular reason to rename this?

Also `sidebar` in the function name is redundant the file is only for sidebar, it's apparent already.